.github/workflows/nuget-publish.yml

```
@@ -0,0 +1,70 @@
+name: NuGet Publish
+
+on:
+  push:
+    branches: [ master ]
+
+env:
+  NUGET_DIR: ${{ github.workspace }}/nuget
+  NUGET_ARTIFACT: nuget
+
+jobs:
+  testing:
+    if: contains(github.event.head_commit.message, 'chore(release)') 
+    uses: ./.github/workflows/test.yml
+
+  pack: 
+    if: contains(github.event.head_commit.message, 'chore(release)') 
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v3
+
+      - run: sed -E 's:RepositoryCommit>.+<\/:RepositoryCommit>${{ github.sha }}<\/:g' ButterCMS/ButterCMS.csproj > tmp && mv tmp ButterCMS/ButterCMS.csproj
```

- You don't need to escape `/` character (`\/` in `RepositoryCommit>.+<\/`) if you used `:` for the delimiter.
- You can edit files in-place using `-I` option.
```suggestion
      - run: sed -E -I'.bak' 's:RepositoryCommit>.+</:RepositoryCommit>${{ github.sha }}</:g' ButterCMS/ButterCMS.csproj ButterCMS/ButterCMS.csproj
```

---

> You don't need to escape / character (\/ in RepositoryCommit>.+<\/) if you used : for the delimiter.

@Toreno96  I didn't work without escape on Mac 😢 . But it doesn't matter since in CI.

---

Weird, it worked on my Mac 😕 But yeah, that will run inside CI/CD, so it's not an issue. For the future, you could test using [gsed](https://formulae.brew.sh/formula/gnu-sed) or run Ubuntu in Docker if something's not working on your Mac.